### PR TITLE
Improve loader for vs code

### PR DESCRIPTION
**Pull request by carlitux**
If language is an array process all of them

### CODE REVIEW

**Review by leiserfg — APPROVED** [review]
Thanks, I was planing to fix that today.

**Comment by leiserfg** [comment]
@carlitux  prefixes (aka triggers) can also be repeated.

File: lua/loaders/from_vscode.lua [hunk 1]
```diff
@@ -41,23 +41,26 @@ local function file_read(path)
...
+            local body = type(parts.body) == "string" and parts.body or table.concat(parts.body, '\n')
+
+            -- There are still some snippets that fail while loading
+            pcall(function()
+            table.insert(
+                lang_snips,
+                ls.parser.parse_snippet({trig=parts.prefix, name=name, wordTrig=true}, body)
+            )
+            end) 
+        end
```
Comment (L3MON4D3) [inline lua/loaders/from_vscode.lua:61]: First of all, thanks :D
Second, are you certain that wrapping `table.insert` into a function is necessary? Am I missing something? :sweat_smile:

Comment (leiserfg) [reply lua/loaders/from_vscode.lua:61]: Yes, the parser can fail if the snippet is wrong.

Comment (L3MON4D3) [reply lua/loaders/from_vscode.lua:61]: Yeah, I agree with the `pcall`, I just think `function() table.insert(...) end` can be simplified into `table.insert(...)`.

Comment (leiserfg) [reply lua/loaders/from_vscode.lua:61]: pcall needs a function as parameter

Comment (L3MON4D3) [reply lua/loaders/from_vscode.lua:61]: Ah I see now, my mistake, thought pcall also error-checks the args but it doesn't.

**Pull request merged**